src/overlays/actors/ovl_En_GirlA/z_en_girla.c

```diff
@@ -202,9 +202,9 @@ static ShopItemEntry shopItemEntries[] = {
     /* SI_ZORA_TUNIC */
     { OBJECT_GI_CLOTHES, GID_TUNIC_ZORA, NULL, 300, 1, 0x00AB, 0x0094, GI_TUNIC_ZORA, EnGirlA_CanBuy_ZoraTunic,
       EnGirlA_ItemGive_ZoraTunic, EnGirlA_BuyEvent_ZoraTunic },
-    /* SI_HEART */
-    { OBJECT_GI_HEART, GID_HEART, NULL, 10, 16, 0x00AC, 0x0095, GI_HEART, EnGirlA_CanBuy_Health,
-      EnGirlA_ItemGive_Health, EnGirlA_BuyEvent_ShieldDiscount },
+    /* SI_RECOVERY_HEART */
+    { OBJECT_GI_HEART, GID_RECOVERY_HEART, NULL, 10, 16, 0x00AC, 0x0095, GI_RECOVERY_HEART,
+      EnGirlA_CanBuy_RecoveryHeart, EnGirlA_ItemGive_Health, EnGirlA_BuyEvent_ShieldDiscount },
```

Comment:
```suggestion
    /* SI_RECOVERY_HEART */
    { OBJECT_GI_HEART, GID_RECOVERY_HEART, NULL, 10, 16, 0x00AC, 0x0095, GI_RECOVERY_HEART,
      EnGirlA_CanBuy_Health, EnGirlA_ItemGive_Health, EnGirlA_BuyEvent_ShieldDiscount },
```
Double checked the code. I probably opted for `EnGirlA_CanBuy_Health` here instead of EnGirlA_CanBuy_RecoveryHeart because a shop item entry can be configured to give more or less than a recovery heart's standard 16 health without any modification to the functions `EnGirlA_CanBuy_Health` and `EnGirlA_ItemGive_Health` (item count configures how much health you receive. If you disagree, `EnGirlA_ItemGive_Health` should still be renamed.

Reply:
Note `EnGirlA_ItemGive_Health` is not an ItemGive at all, it could do `Item_Give(play, ITEM_HEART);` but doesn't, and changes the health directly instead. Of course it should stay named like `EnGirlA_ItemGive_` since it's a `ShopItemEntry.itemGiveFunc`, we just have to deal with that inconsistency

So I think `EnGirlA_ItemGive_Health` is fine for the reason you mentioned, it isn't necessarily one heart

-----------

For `EnGirlA_CanBuy_Health`/`EnGirlA_CanBuy_RecoveryHeart` it's whatever to me, both work.

`EnGirlA_CanBuy_Health` because:
- consistency with `EnGirlA_ItemGive_Health`

`EnGirlA_CanBuy_RecoveryHeart` because:
- consistency with `SI_RECOVERY_HEART`